### PR TITLE
Add to-readable-stream type definitions

### PR DESCRIPTION
**Pull request by Woo-Yong0405**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/to-readable-stream/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Type definitions for url-parse-lax 5.0
+// Project: https://github.com/sindresorhus/url-parse-lax
+// Definitions by: Woo Yong Seok <https://github.com/woo-yong0405>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Readable } from "stream";
+
+export default function toReadableStream(input: string | Uint8Array): Readable;
```
Comment (peterblazejewicz) [inline types/to-readable-stream/index.d.ts:10]:
```ts
// Type definitions for url-parse-lax 5.0
// Project: https://github.com/sindresorhus/url-parse-lax
// Definitions by: Woo Yong Seok <https://github.com/woo-yong0405>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped

/// <reference types="node" />

import { Readable } from "stream";

export default function toReadableStream(input: string | Uint8Array): Readable;
```
this looks wrong, please review which module you're adding, for now it looks like it's subject is `url-parse-las` module.

Comment (Woo-Yong0405) [reply types/to-readable-stream/index.d.ts]: Done. Thank you for the reply.

**Comment by peterblazejewicz** [comment]
OK, now when this is settled, the package you're trying to add, seems to already shipping with it's own type definition:
https://unpkg.com/browse/to-readable-stream@3.0.0/index.d.ts

**Comment by Woo-Yong0405** [comment]
Oh. So I just wasted 2 hours of my life lol

**Comment by peterblazejewicz** [comment]
sorry to hear, take a note about NPM registry, from some time it shows information about TS support for module:
image

those can be 'TS' or 'DT', depending on native support or support with declaration files,
thanks!

**Pull request closed without merging**